extensions/amp-facebook/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+    }
+  })();
+</script>
+
+<!-- Facebook Comment-->
+<amp-facebook id="facebook-comment" width="552" height="500"
+    layout="responsive"
+    data-embed-type="comment"
+    data-href="https://www.facebook.com/zuck/posts/10102735452532991?comment_id=1070233703036185">
+</amp-facebook>
+<div class="buttons" style="margin-top: 8px;">
+  <button id="change-facebook-comment">
+    Change Facebook comment
+  </button>
+</div>
+
+<script>
+  (async () => {
+    const facebookComment= document.querySelector('#facebook-comment');
+    await customElements.whenDefined('bento-facebook');
+    // set up button actions
+    document.querySelector('#change-facebook-comment').onclick = () => {
+      facebookComment.setAttribute('data-href', 'https://www.facebook.com/NASA/photos/a.67899501771/10159193669016772/?comment_id=10159193676606772')
+    }
+  })();
+</script>
+```
+
+[/example]
+
+#### Layout and Style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-facebook-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  bento-facebook {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
+```
+
+#### Attributes
+
+##### `data-href`
+
+The URL of the Facebook post/video/comment. For example, a post or video will
+look like `https://www.facebook.com/zuck/posts/10102593740125791`. A comment or
+comment reply will look like
+`https://www.facebook.com/zuck/posts/10102735452532991?comment_id=1070233703036185`.
+For comments, see the Facebook documentation on
+[how to get a comment's URL](https://developers.facebook.com/docs/plugins/embedded-comments#how-to-get-a-comments-url).
+
+##### `data-embed-as`
+
+The value is either `post`, `video` or `comment`. The default is `post`.
+
+Both posts and videos can be embedded as a post. Setting `data-embed-as="video"`
+for Facebook videos embeds the player of the video, and adds the accompanying
+post card with it. Setting `data-embed-as="post"` ignores the caption card. This
+is done to make sure we are zooming in on videos correctly.
+
+The `comment` value embeds a single comment (or reply to a comment) on a post.
+This is not to be confused with
+[`amp-facebook-comments`](https://amp.dev/documentation/examples/components/amp-facebook-comments/).
+
+Check out the documentation for differences between
+[post embeds](https://developers.facebook.com/docs/plugins/embedded-posts),
+[video embeds](https://developers.facebook.com/docs/plugins/embedded-video-player),
+and [comment embeds](https://developers.facebook.com/docs/plugins/embedded-comments).
```

Comment:
There should be six options total: `post`, `video`, `comment`, `comments`, `like`, `page`